cruise-control/src/yaml/responses/rightsizeResult.yaml

```diff
@@ -0,0 +1,25 @@
+RightsizeResult:
+  type: object
+  required:
+    - numBrokersToAdd
+    - partitionCount
+    - topic
+    - provisionerState
+    - provisionerSummary
+    - version
```

Comment:
Looks like we listed all fields as `required`, but are all of them guaranteed to be in the response?
For example, based on `RightsizeResult#getJSONString()`, unless the topic contains characters it is not returned in the response.

Can we update this list to contain only the required items?
You may check https://swagger.io/docs/specification/basic-structure/ for details on what each keyword means.

Reply:
Updating to set only the version as required